packages/fast-components-styles-msft/src/utilities/color/accent-foreground-cut.ts

```diff
@@ -1,10 +1,9 @@
 import { Swatch } from "./palette";
 import { black, white } from "./color-constants";
-import chroma from "chroma-js";
```

Comment:
can you not remove chroma-js from the package.json now?

Reply:
not yet - there are still utilities that reference it.